File: Estrutura de Repeticao/Exercicio 2/Eleicao.php
<?php
/*
Numa eleição existem três candidatos. Faça um programa que peça o número total de eleitores. Peça para cada eleitor votar e
ao final mostrar o número de votos de cada candidato.*/

$candidatoA = intval(0);
$candidatoB = intval(0);
$candidatoC = intval(0);
$nulos = intval(0);
$eleitores = intval(readline("Qual o número de eleitores?"));
for ($i = 1; $i < $eleitores + 1; $i++) {
    $voto = intval(
        readline("Eleitor " . $i . ", vote em um candidato:"
            . PHP_EOL . "[1] Candidato A" . PHP_EOL .
            "[2] Candidato B " .
            PHP_EOL . "[3] Candidato C")
    );
    if ($voto == 1) {
        $candidatoA++;
    } else if ($voto == 2) {
        $candidatoB++;
    } else if ($voto == 3) {
        $candidatoC++;
    } else {
        echo "Você escolheu um inválido. Seu voto será nulo.";
        $nulos++;
    }
}

echo "RESULTADO:" . PHP_EOL;
echo "Candidato A: " . $candidatoA . PHP_EOL;
echo "Candidato B: " . $candidatoB . PHP_EOL;
echo "Candidato C: " . $candidatoC . PHP_EOL;
echo "Nulos: " . $nulos . PHP_EOL;

if ($candidatoA > $candidatoB && $candidatoA > $candidatoC) {
    echo "Candidato A foi o vencedor." . PHP_EOL;
} elseif ($candidatoB > $candidatoA && $candidatoB > $candidatoC) {
    echo "Candidato C foi o vencedor." . PHP_EOL;
} elseif ($candidatoC > $candidatoA && $candidatoC > $candidatoB) {
    echo "Candidato B foi o vencedor." . PHP_EOL;
} else {
    echo "Houve empate. Uma nova eleição será necessária." . PHP_EOL;
}

File: Estrutura de Repeticao/Exercicio 2/Eleicao.py
'''
Numa eleição existem três candidatos. Faça um programa que peça o número total de eleitores. Peça para cada eleitor votar e
ao final mostrar o número de votos de cada candidato.
'''
import os

candidato_a = int(0)
candidato_b = int(0)
candidato_c = int(0)
nulos = int(0)
eleitores = int(input("Qual o número de eleitores?\n"))
os.system('cls')
i = 1
for i in range(1, eleitores + 1):
    os.system("cls")
    voto = input(
        f"Eleitor {i}. Vote em um candidato:\n[1] Candidato A\n[2] Candidato B\n[3] Candidato C\n")
    voto = int(voto)
    if voto == 1:
        candidato_a += 1
    elif voto == 2:
        candidato_b += 1
    elif voto == 3:
        candidato_c += 1
    else:
        print("Você escolheu um inválido. Seu voto será nulo.")
        nulos += 1

os.system("cls")

print("RESULTADO:\n")
print(f'Candidato A: {candidato_a}\n')
print(f'Candidato B: {candidato_b}\n')
print(f'Candidato C: {candidato_c}\n')
if candidato_a > candidato_b and candidato_a > candidato_c:
    print("Candidato A foi o Vencedor.")
elif candidato_b > candidato_a and candidato_b > candidato_c:
    print("Candidato B foi o Vencedor.")
elif candidato_c > candidato_a and candidato_c > candidato_b:
    print("Candidato C foi o Vencedor.")
else:
    print("Houve empate. Uma nova eleição será necessária.")
